rest-fights/openapi-schema.yml

```diff
@@ -72,15 +60,14 @@ paths:
   /api/fights/{id}:
     get:
       tags:
-      - fights
+        - fights
       summary: Returns a fight for a given identifier
       parameters:
-      - name: id
-        in: path
-        required: true
-        schema:
-          format: int64
-          type: integer
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
```

Comment:
Looks like you found something I missed in a previous change :) 👏

Reply:
Yes. MongoDB ids are String. I suppose the integer was when it was PostgreSQL